blog/docs/articles/knative-meets-apache-camel.md

```diff
@@ -0,0 +1,96 @@
+# Event Sourcing with Apache Camel K and Knative Eventing
+
+**Author: Matthias Weßendorf, Senior Principal Software Engineer @ Red Hat**
+
+## Why Apache Camel K?
+
+The [Apache Camel](https://camel.apache.org/){:target="_blank"} is a popular Open Source integration framework that empowers you to quickly and easily integrate various systems consuming or producing data. With [Apache Camel K](https://camel.apache.org/camel-k/latest){:target="_blank"} the project provides a lightweight integration framework built from Apache Camel that runs natively on Kubernetes and is specifically designed for serverless and microservice architectures.
+
+The Camel K framework does also support Knative, allowing developers to [bind](https://camel.apache.org/camel-k/latest/kamelets/kamelets-user.html#kamelets-usage-binding){:target="_blank"} any Kamelet to a Knative component. A Kamelet can act as "source" of data or alternatively as "sink". There are several Kamelets available for integrating and connecting to 3rd party services or products, such as Amazon Web Services (AWS), Google Cloud or even tradition message systems like AMQP 1.0 or JMS brokers like Apache Artemis. The full list of Kamelets can be found [here](https://camel.apache.org/camel-kamelets/latest/index.html){:target="_blank"}.
+
+## Installation
+
+The [Installation](https://camel.apache.org/camel-k/next/installation/installation.html) from Apache Camel K offers a few choices, such as CLI, Kustomize, OLM or Helm, like:
+
+```
+$ helm repo add camel-k https://apache.github.io/camel-k/charts/
+$ helm install my-camel-k camel-k/camel-k
+```
+
+Beside Camel K we also need to have Knative Eventing installed, as discussed [here](https://knative.dev/docs/install/yaml-install/eventing/install-eventing-with-yaml/){:target="_blank"}.
+
+## Creating a Knative Broker instance
+
+We are using a Knative Broker as the heart of our system, acting as an [Event Mesh](https://knative.dev/docs/eventing/event-mesh/){:target="_blank"} for both event producers and event consumers:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  namespace: default
+  name: demo-broker
+```
+
+Now event producers can send events to it and event consumers can receive events.
+
+## Using Kamelets as Event Sources
+
+In order to bind a Kamelet to a Knative component, like the above broker, we are using the `Pipe` API. A Pipe allows to declaratively move data from a system described by a Kamelet _towards_ a Knative destination **or** _from_ a Knative destination to another (external) system described by a Kamelet.
+
+Below is a `Pipe` that uses a ready-to-use `Kamelet`, the `timer-source`
+
+```yaml
+apiVersion: camel.apache.org/v1
+kind: Pipe
+metadata:
+  name: beer-source-pipe
+spec:
+  source:
+    ref:
+      kind: Kamelet
+      apiVersion: camel.apache.org/v1
+      name: timer-source
+    properties:
```

Comment:
Maybe you add a quick note, that the `properties` field is specific for the referenced kind (from .spec.source.ref). As you later use the same for the Trigger filter (line 81 & 82)

Reply:
I have a sentence below that example.

Reply:
:+1: